ci_framework/roles/libvirt_manager/templates/kustomize_compute.yml.j2

```diff
@@ -0,0 +1,28 @@
+{%- set compute_range = range(0, compute_config.amount | default(1) | int) | list -%}"
```

Comment:
May be we can check edpm-compute-(compute_config.amout -1) node exists in the cr file and then apply the kustomization. this way we can avoid failing.